components/AddFundsBtn.js

```diff
@@ -11,13 +11,7 @@ const AddFundsBtn = ({ children, collective, host }) => {
     <Fragment>
       {children({ onClick: () => setShowModal(true) })}
       {showModal && (
-        <AddFundsModal
-          show
-          collective={collective}
-          host={host}
-          setShow={setShowModal}
-          onClose={() => setShowModal(null)}
-        />
+        <AddFundsModal collective={collective} host={host} setShow={setShowModal} onClose={() => setShowModal(null)} />
```

Comment:
Similarily to https://github.com/opencollective/opencollective-frontend/pull/7518#discussion_r814548450, it's not from this PR but we may as well clean it up while we're there

```suggestion
        <AddFundsModal collective={collective} host={host} onClose={() => setShowModal(null)} />
```

Reply:
since it's not really part of the original issue, I put it in a separate commit. But it's a very small refactor, so I guess no need for separate PR.